index.js

```diff
@@ -23,16 +23,23 @@ function getHash(data) {
  * @param {string} src source of the file being compiled to ES5
  * @return {string} compiled source
  */
-function compileFile(file, src) {
+function compileFile(file, src, opts) {
   var compiled;
-  compiled = compile(file, src, exports.traceurOverrides);
+  opts.traceurOverrides = exports.traceurOverrides;
+  compiled = compile(file, src, opts);
   if (compiled.error) throw new Error(compiled.error);
 
   return compiled.source;
 }
 
-function es6ify(filePattern) {
-  filePattern =  filePattern || /\.js$/;
+function es6ify(opts) {
+  opts || (opts = {});
```

Comment:
> Also best to follow ES6 and do if (opts === undefined) { opts = {}; } instead of using any falsy value.
> Not ES6ey enough, only undefined should trigger defaults :)

I don't see what it accomplishes -- the new test will still allow a bunch of nonsensical values, and convert `es6ify(null)` (which admittedly doesn't make sense) to a `TypeError` -- but it's up to you guys. Also, if making this codebase ES6ey is an imperative, why not actually author it in ES6 and transpile it? 

> Please use a proper `if` or `?:`

You're referring to readability preferences not functionality, right? A `?:` in this situation seems like the worst option to me.